### PR TITLE
add : agus ViewBeans and others

### PR DESCRIPTION
**Pull request by AlucardTN**
again, TESTED && WORKING

### CODE REVIEW

File: app/src/main/java/mod/agus/jcoderz/editor/view/item/ItemTimePicker.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.view.MotionEvent;
+import android.widget.TimePicker;
+import com.besome.sketch.beans.ViewBean;
+
+public class ItemTimePicker extends TimePicker implements sy {
+
+
+    public ViewBean f23a;
```
Comment (JavkhlanK) [inline app/src/main/java/mod/agus/jcoderz/editor/view/item/ItemTimePicker.java:16]: This one too.

File: app/src/main/java/mod/agus/jcoderz/editor/view/item/ItemDatePicker.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.view.MotionEvent;
+import android.widget.DatePicker;
+import com.besome.sketch.beans.ViewBean;
+
+public class ItemDatePicker extends DatePicker implements sy {
+
+
+    public ViewBean f21a;
```
Comment (JavkhlanK) [inline app/src/main/java/mod/agus/jcoderz/editor/view/item/ItemDatePicker.java:16]: Please rename this field back to `a`.

File: app/src/main/java/mod/agus/jcoderz/editor/view/item/ItemGridView.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+import android.widget.ArrayAdapter;
+import android.widget.GridView;
+import android.widget.ListAdapter;
+import com.besome.sketch.beans.ViewBean;
+import java.util.ArrayList;
+
+public class ItemGridView extends GridView implements sy {
+
+
+    public ViewBean f22a;
```
Comment (JavkhlanK) [inline app/src/main/java/mod/agus/jcoderz/editor/view/item/ItemGridView.java:20]: Rename this one back to `a` too, please.

**Comment by MarkDroid** [comment]
bru again, they are working, how much i need to say that, i tested with all widgets and everything is 101% fine

**Comment by MarkDroid** [comment]
im using rename-flags none in jadx, and even with rename-flags or not i get the same name of the fields

**Comment by AlucardTN** [comment]
merger the pull or dumb

**Pull request merged**